code/index.html

```diff
@@ -2,21 +2,92 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>classic newspaper</title>
     <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+ 
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <header>        
+       <section class="header-container" >
+          <img src="divider-4715969.svg" alt="devider">
+          <h1>Classic Newspaper</h1>
+        </section>
+        <section class="the-first-container">
+          <div class="hamburger"> 
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+          <nav class=nav>
+            <a href="#" ></a>World</a>
```

Comment:
You are ending the `a`-tag before the text
```suggestion
            <a href="#">World</a>
```